### PR TITLE
fwupd: update to 1.3.3

### PR DESCRIPTION
**Pull request by ndowens**

### CODE REVIEW

File: srcpkgs/tpm2-tss/template [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+pre_configure() {
+	./bootstrap
+}
+
+post_install() {
+	vlicense LICENSE
+}
+
+tpm2-tss-devel_package() {
+	depends="${sourcepkg}>=${version}_${revision}"
```
Comment (xtraeme) [inline srcpkgs/tpm2-tss/template]: I guess this is incomplete. anyway why do you use libgcrypt?

Comment (ndowens) [reply srcpkgs/tpm2-tss/template]: It wants to use OpenSSL and leave it to try and use LibreSSL it gives compilation issues, like RAND_OpenSSL undefined references.

Incomplete as in?

File: common/shlibs [hunk 2]
```diff
@@ -16,6 +16,13 @@
 # PLEASE NOTE: when multiple packages provide the same SONAME, the first
 # one (order top->bottom) is preferred over the next ones.
 #
+libtss2-mu.so.0 tpm2-tss-2.3.1_1
```
Comment (sgn) [inline common/shlibs]: Those so maybe important to either _you_ or fwupd.
But it's not as important as other package, especially libc.

Better to move it down the list, last of the list should be better.

File: srcpkgs/tpm2-tss/template (hunk 1 above)
Comment (sgn) [reply srcpkgs/tpm2-tss/template]: Does it need `depends+="$makedepends"`?

File: common/shlibs (hunk 2 above)
Comment (ndowens) [reply common/shlibs]: I'll move it down in next revision

**Comment by ndowens** [comment]
Unsure on the depends+=$makedepends Every subpackage I seen used depends the way I have it.

File: srcpkgs/tpm2-tss/template (hunk 1 above)
Comment (ndowens) [reply srcpkgs/tpm2-tss/template]: Unsure on the depends+=$makedepends Every subpackage I seen used depends the way I have it.

Comment (sgn) [reply srcpkgs/tpm2-tss/template]: Checked its header, it doesn't need `makedepends`

Comment (xtraeme) [inline srcpkgs/tpm2-tss/template]: Take a look at its pkgconfig files for additional dependencies...

Comment (ndowens) [reply srcpkgs/tpm2-tss/template]: From what I seen when looking at them, they depend on parts of the package itself, except one stated gcrypt

**Pull request closed without merging**